bluechi.spec.in

```diff
@@ -353,6 +353,9 @@ build-scripts/generate-unit-tests-code-coverage.sh %{_vpath_builddir} %{buildroo
 
 
 %changelog
+* Wed May 28 2025 Michael Engel <mengel@redhat.com> - 1.0.1-1
+- Fxied meson parameters for subproject inih
```

Comment:
I'd change to `Fixed parameters for subproject inih to support meson 1.8`

Reply:
Sounds good. Changed :heavy_check_mark: